openforis/collect-earth-online
# CEO-411 Rename survey rules keys for consistency

## Purpose
Rename survey rules keys for consistency.

## Submission Checklist

## Testing
The code should work as it has.  The existing BDD testing should work, plus:
1. load a question tree with questions and answer
2. Add a rule to a parent and a child question
3. The parent and child question should show a rule icon in the survey collection preview.

![image](https://user-images.githubusercontent.com/33734037/147996308-de7329d6-1ea0-45fa-a54c-b987f66ca785.png)
